src/javascript/app/Stores/Modules/Trading/trade-store.js

```diff
@@ -575,22 +575,36 @@ export default class TradeStore extends BaseStore {
 
     @action.bound
     async onMount() {
+        this.onLoadingMount();
         await this.prepareTradeStore();
         this.debouncedProposal();
         runInAction(() => {
             this.is_trade_component_mounted = true;
         });
-        this.onLoadingMount();
         this.updateQueryString();
         this.onSwitchAccount(this.accountSwitcherListener);
     }
 
     @action.bound
     onLoadingMount() {
-        // TODO: find better way to remove initial loader
-        setTimeout(() => {
-            this.root_store.ui.setAppLoading(false);
-        }, 2200);
+        const first_timeout = setTimeout(() => {
+            const loading_message = localize('Your network connection might be slow, Please wait for the page to finish loading.');
```

Comment:
`Please` doesn't need capitalization after `comma`. Maybe we should get copywriter's opinion/approval for these messages?

Reply:
yes, the words are provided by reuben, i will confirm with him

Reply:
he said, it should be `.` fullstop instead of comma